languages/javascript/config.json

```
@@ -28,16 +29,21 @@
         "prerequisites": []
       },
       {
-        "slug": "arrays",
-        "uuid": "25cb0707-44f8-4800-b993-3fcb2b6d9f61",
-        "concepts": ["arrays"],
-        "prerequisites": ["numbers"]
+        "slug": "numbers",
+        "uuid": "0aff2fa7-55ea-47e9-af4a-78927d916baf",
+        "concepts": ["numbers"],
+        "prerequisites": []
```

I think you want to have the "basics" concept as a prerequisite?

```suggestion
        "prerequisites": ["basics"]
```